### PR TITLE
Add confirm dialog on delete actions

### PR DESCRIPTION
**Pull request by Yanovskiy**
Add confirm dialog modal on delete actions:
 - users
 - accounts
 - storage

### CODE REVIEW

File: console/ui/src/app/accounts/accounts.component.ts [hunk 1]
```diff
@@ -110,16 +110,18 @@ export class AccountListComponent implements OnInit {
   }
 
   deleteAccount(event, i: number, o: ApiUser): void {
-    event.target.disabled = true;
-    event.preventDefault();
-    this.error = '';
-    this.consoleService.deleteAccount('', o.id, false).subscribe(() => {
+    if (confirm('Are you sure to delete ' + o.username + '. All associated data will be lost')) {
```
Comment (sesposito) [inline console/ui/src/app/accounts/accounts.component.ts:113]: The use of js confirm dialogs is discouraged as they cannot be styled and won't be consistent with the existing confirmation prompts we already have elsewhere (see: https://github.com/heroiclabs/nakama/blob/master/console/ui/src/app/config/config.component.html#L97).

File: console/ui/src/app/storage/storage.component.ts [hunk 2]
```diff
@@ -125,16 +125,18 @@ export class StorageListComponent implements OnInit {
   }
 
   deleteObject(event, i: number, o: ApiStorageObject): void {
-    event.target.disabled = true;
-    event.preventDefault();
-    this.error = '';
-    this.consoleService.deleteStorageObject('', o.collection, encodeURIComponent(o.key), o.user_id, o.version).subscribe(() => {
+    if (confirm('Are you sure to delete ' + o.key + ' from ' + o.collection + '?')) {
```
Comment (sesposito) [inline console/ui/src/app/storage/storage.component.ts:128]: ditto

File: console/ui/src/app/users/users.component.ts [hunk 3]
```diff
@@ -59,17 +59,19 @@ export class UsersComponent implements OnInit {
   }
 
   public deleteUser(username: string): void {
-    this.error = '';
-
-    this.consoleService.deleteUser('', username).pipe(mergeMap(() => {
-      return this.consoleService.listUsers('');
-    })).subscribe(userList => {
+    if (confirm('Are you sure to delete ' + username)) {
```
Comment (sesposito) [inline console/ui/src/app/users/users.component.ts:62]: ditto

**Review by sesposito — CHANGES_REQUESTED** [review]
Thank for your contribution @Yanovskiy, I think the addition of these confirmation dialogs can be a good idea, but instead of js dialogs it should be reworked to use the bootstrap modals for consistency. If you make these changes, please run the steps in the `console/ui` folder as well to package the console js code and include it into the PR.

**Comment by zyro** [comment]
Closing as not mergeable in its current state. If the changes requested in the review can be made, we'd welcome a fresh PR! 👍

**Pull request closed without merging**